recipes/gnupg/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "gnupg" %}
+{% set version = "2.2.4" %}
+{% set sha256 = "401a3e64780fdfa6d7670de0880aa5c9d589b3db7a7098979d7606cec546f2ec" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://www.gnupg.org/ftp/gcrypt/gnupg/gnupg-{{ version }}.tar.bz2
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script:
+    - export CPPFLAGS="-DIN_EXCL_UNLINK=0x04000000 $CPPFLAGS"  # [linux]
+    - export LDFLAGS="-lrt $LDFLAGS"                           # [linux]
+    - ./configure --prefix=$PREFIX
+    - make
+    - make install
+  skip: true  # [win]
+  run_exports:
+    - gnupg
```

Comment:
This should be `- {{ pin_subpackage('zlib', max_pin='x.x') }}` if you assume that there's no ABI changes in patch version

Reply:
the default is 'x' right? so major

Reply:
`- {{ pin_subpackage('zlib') }}` pins to major. `- gnupg` pins downstream to the package with no version.

Reply:
Edited.